src/pages/LandingPage.vue

```diff
@@ -1,25 +1,52 @@
 <template>
-  <div class="grid m-auto">
-    <div class="">
-      <p v-if="isLocaleSelectedEnglish" class="text-4xl">Welcome to Portal!</p>
-      <p v-if="isLocaleSelectedHindi" class="text-4xl">
-        पोर्टल में आपका स्वागत है!
-      </p>
-    </div>
+  <div class="flex flex-col min-h-screen">
+    <nav class="p-4 flex justify-between items-center">
+      <div class="text-left">
+        <h1 class="text-2xl font-bold text-blue-600">Avanti Fellows</h1>
+        <p class="text-sm text-gray-600">Enabling Equity in Education</p>
+      </div>
+      <button
+        @click="toggleLocale"
+        class="px-4 py-2 text-sm font-medium text-gray-700 bg-white border border-gray-300 rounded-md shadow-sm hover:bg-gray-50 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500"
+      >
+        {{ isEnglish ? 'हिंदी' : 'English' }}
```

Comment:
no -- should be based on dropdown selection, not a state variable that doesnt change